### PR TITLE
dcinside.com - Ad

### PR DESCRIPTION
**Pull request by green1052**

### CODE REVIEW

**Review by piquark6046 — CHANGES_REQUESTED** [review]
Do these requests appear in only dcinside mobile app?

**Comment by green1052** [comment]
`||dcinside.com/_js/ad_keyword_powLink.js`
i found it during search 

`||json*.dcinside.com/json*/app_ad_charge.php`
only mobile app

**Comment by piquark6046** [comment]
@green1052 Please move `||json*.dcinside.com/json*/app_ad_charge.php` rule into `/filters-AG/mobile/android/android_specific.txt`.

**Comment by green1052** [comment]
remove comment? @piquark6046

**Comment by piquark6046** [comment]
Which the comment?
Please attach its URL address.

**Comment by green1052** [comment]
> Which the comment? Please attach its URL address.

`
! Rules of this file are compatible rules for only an issue that can be reproduced with AdGuard for Android.
! If a website, mobile version, can be reproduced on a desktop/laptop browser with AdGuard for Windows/AdGuard Browser Extension,
! please move into specific.txt.
`

**Comment by piquark6046** [comment]
Nope. Just add it into its below.

**Comment by green1052** [comment]
done

File: filters-AG/mobile/android/android_specific.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ! Rules of this file are compatible rules for only an issue that can be reproduced with AdGuard for Android.
 ! If a website, mobile version, can be reproduced on a desktop/laptop browser with AdGuard for Windows/AdGuard Browser Extension,
-! please move into specific.txt.
\ No newline at end of file
+! please move into specific.txt.
+
```
Comment (piquark6046) [inline filters-AG/mobile/android/android_specific.txt]: Without an empty line.

**Review by piquark6046 — CHANGES_REQUESTED** [review]

**Review by piquark6046 — APPROVED** [review]

**Pull request merged**